<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - Class HiddenTests</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>Class HiddenTests</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/default-package.html">default-package</a> &gt; HiddenTests</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">7</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">6</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">0.344s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox failures" id="successRate">
<div class="percent">14%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Failed tests</a>
</li>
<li>
<a href="#tab1">Tests</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Failed tests</h2>
<div class="test">
<a name="randomLarge()"></a>
<h3 class="failures">randomLarge()</h3>
<span class="code">
<pre>org.opentest4j.AssertionFailedError: [length(s) = 16929] incorrect number of blocks ==&gt; expected: &lt;4262&gt; but was: &lt;8445&gt;
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:69)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:156)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:594)
	at app//HiddenTests.testSingle(HiddenTests.kt:128)
	at app//HiddenTests.testRandom(HiddenTests.kt:80)
	at app//HiddenTests.access$testRandom(HiddenTests.kt:9)
	at app//HiddenTests$randomLarge$1.invoke(HiddenTests.kt:65)
	at app//HiddenTests$randomLarge$1.invoke(HiddenTests.kt:64)
	at app//org.junit.jupiter.api.AssertionsKt$sam$org_junit_jupiter_api_function_ThrowingSupplier$0.get(Assertions.kt)
	at app//org.junit.jupiter.api.AssertTimeout.lambda$assertTimeoutPreemptively$4(AssertTimeout.java:138)
	at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
</pre>
</span>
</div>
<div class="test">
<a name="randomMax()"></a>
<h3 class="failures">randomMax()</h3>
<span class="code">
<pre>org.opentest4j.AssertionFailedError: [length(s) = 997455] incorrect number of blocks ==&gt; expected: &lt;249028&gt; but was: &lt;499238&gt;
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:69)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:156)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:594)
	at app//HiddenTests.testSingle(HiddenTests.kt:128)
	at app//HiddenTests.testRandom(HiddenTests.kt:80)
	at app//HiddenTests.access$testRandom(HiddenTests.kt:9)
	at app//HiddenTests$randomMax$1.invoke(HiddenTests.kt:71)
	at app//HiddenTests$randomMax$1.invoke(HiddenTests.kt:70)
	at app//org.junit.jupiter.api.AssertionsKt$sam$org_junit_jupiter_api_function_ThrowingSupplier$0.get(Assertions.kt)
	at app//org.junit.jupiter.api.AssertTimeout.lambda$assertTimeoutPreemptively$4(AssertTimeout.java:138)
	at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
</pre>
</span>
</div>
<div class="test">
<a name="randomMedium()"></a>
<h3 class="failures">randomMedium()</h3>
<span class="code">
<pre>org.opentest4j.AssertionFailedError: [length(s) = 123] incorrect number of blocks ==&gt; expected: &lt;28&gt; but was: &lt;66&gt;
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:69)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:156)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:594)
	at app//HiddenTests.testSingle(HiddenTests.kt:128)
	at app//HiddenTests.testRandom(HiddenTests.kt:80)
	at app//HiddenTests.access$testRandom(HiddenTests.kt:9)
	at app//HiddenTests$randomMedium$1.invoke(HiddenTests.kt:59)
	at app//HiddenTests$randomMedium$1.invoke(HiddenTests.kt:58)
	at app//org.junit.jupiter.api.AssertionsKt$sam$org_junit_jupiter_api_function_ThrowingSupplier$0.get(Assertions.kt)
	at app//org.junit.jupiter.api.AssertTimeout.lambda$assertTimeoutPreemptively$4(AssertTimeout.java:138)
	at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
</pre>
</span>
</div>
<div class="test">
<a name="randomMediumInstances()"></a>
<h3 class="failures">randomMediumInstances()</h3>
<span class="code">
<pre>org.opentest4j.AssertionFailedError: [length(s) = 80] incorrect number of blocks ==&gt; expected: &lt;24&gt; but was: &lt;43&gt;
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:69)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:156)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:594)
	at app//HiddenTests.testSingle(HiddenTests.kt:128)
	at app//HiddenTests.testRandom(HiddenTests.kt:80)
	at app//HiddenTests.access$testRandom(HiddenTests.kt:9)
	at app//HiddenTests$randomMediumInstances$1.invoke(HiddenTests.kt:52)
	at app//HiddenTests$randomMediumInstances$1.invoke(HiddenTests.kt:49)
	at app//org.junit.jupiter.api.AssertionsKt$sam$org_junit_jupiter_api_function_ThrowingSupplier$0.get(Assertions.kt)
	at app//org.junit.jupiter.api.AssertTimeout.lambda$assertTimeoutPreemptively$4(AssertTimeout.java:138)
	at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
</pre>
</span>
</div>
<div class="test">
<a name="randomSmall()"></a>
<h3 class="failures">randomSmall()</h3>
<span class="code">
<pre>org.opentest4j.AssertionFailedError: [length(s) = 19] incorrect number of blocks ==&gt; expected: &lt;6&gt; but was: &lt;8&gt;
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:69)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:156)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:594)
	at app//HiddenTests.testSingle(HiddenTests.kt:128)
	at app//HiddenTests.testRandom(HiddenTests.kt:80)
	at app//HiddenTests.access$testRandom(HiddenTests.kt:9)
	at app//HiddenTests$randomSmall$1.invoke(HiddenTests.kt:45)
	at app//HiddenTests$randomSmall$1.invoke(HiddenTests.kt:44)
	at app//org.junit.jupiter.api.AssertionsKt$sam$org_junit_jupiter_api_function_ThrowingSupplier$0.get(Assertions.kt)
	at app//org.junit.jupiter.api.AssertTimeout.lambda$assertTimeoutPreemptively$4(AssertTimeout.java:138)
	at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
</pre>
</span>
</div>
<div class="test">
<a name="testMax()"></a>
<h3 class="failures">testMax()</h3>
<span class="code">
<pre>org.opentest4j.AssertionFailedError: [length(s) = 1000000] incorrect number of blocks ==&gt; expected: &lt;249877&gt; but was: &lt;500152&gt;
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:69)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:156)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:594)
	at app//HiddenTests.testSingle(HiddenTests.kt:128)
	at app//HiddenTests.access$testSingle(HiddenTests.kt:9)
	at app//HiddenTests$testMax$1.invoke(HiddenTests.kt:76)
	at app//HiddenTests$testMax$1.invoke(HiddenTests.kt:75)
	at app//org.junit.jupiter.api.AssertionsKt$sam$org_junit_jupiter_api_function_ThrowingSupplier$0.get(Assertions.kt)
	at app//org.junit.jupiter.api.AssertTimeout.lambda$assertTimeoutPreemptively$4(AssertTimeout.java:138)
	at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
</pre>
</span>
</div>
</div>
<div id="tab1" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="failures">randomLarge()</td>
<td class="failures">0.005s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">randomMax()</td>
<td class="failures">0.106s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">randomMedium()</td>
<td class="failures">0.004s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">randomMediumInstances()</td>
<td class="failures">0.053s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="failures">randomSmall()</td>
<td class="failures">0.005s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="success">testEmpty()</td>
<td class="success">0.007s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="failures">testMax()</td>
<td class="failures">0.164s</td>
<td class="failures">failed</td>
</tr>
</table>
</div>
</div>
<div id="footer">
<p>
<div>
<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
</label>
</div>Generated by 
<a href="http://www.gradle.org">Gradle 8.0</a> at Sep 14, 2023, 12:13:33 AM</p>
</div>
</div>
</body>
</html>
